### ISSUE TITLE
macos arm64 build for pyin plugin

### ISSUE THREAD

**Issue by gesellkammer** [sonic-visualiser/sonic-visualiser#88]
Hi, trying to actually add an issue to https://code.soundsoftware.ac.uk/projects/pyin/issues but it does not seem to be possible to do it there. Is there a way to provide a native arm64 build of the pyin plugin for macos?

I maintain a fork of the vamp host for python (https://github.com/gesellkammer/vamphost) with a working build for arm64 and would like to test it with native plugins, but would like to avoid the hustle of building those as well.

Thanks!
Eduardo

**Comment by cannam** [comment]
Yes! An ARM Mac build of the Vamp Plugin Pack is pending, as a fairly high priority before the next SV release (which should also be fairly soon I hope). This really should have happened a few years ago, sorry it's been taking so long.
